### PR TITLE
Added a new cool dictionary

### PR DESCRIPTION
**Pull request by Sumit0581**
Added a new cool dictionary "Unicodemix"
#3

### CODE REVIEW

File: services/uncoolifyService.spec.js [hunk 1]
```diff
@@ -75,4 +75,8 @@ describe('uncoolifyService.alphaNumericeName', () => {
   it('should uncoolify doublestruck letters', () => {
     expect(alphaNumericeName('𝕙𝕖𝕝𝕝𝕠')[0]).to.equal('hello');
   });
+
+  it('should uncoolify unicodemix letters', () => {
+    expect(alphaNumericeName('ℌΣ↳↳Φ')[0]).to.equal('hellos');
```
Comment (mohitkyadav) [inline services/uncoolifyService.spec.js]: hellos?

File: services/coolifyService.js [hunk 2]
```diff
@@ -359,9 +387,10 @@ coolify.alphaNumericName = function(name) {
 		let cool_name_curr = name.allReplace(coolifyDicts.curr);
 		let cool_name_fairy = name.allReplace(coolifyDicts.fairy);
 		let cool_name_dot = name.allReplace(coolifyDicts.dot);
-    let cool_name_doublestruck = name.allReplace(coolifyDicts.doublestruck);
+    		let cool_name_doublestruck = name.allReplace(coolifyDicts.doublestruck);
```
Comment (mohitkyadav) [inline services/coolifyService.js]: Fix the indentation.

File: server.js [hunk 3]
```diff
@@ -37,7 +37,8 @@ app.get("/coolify", (request, response) => {
 		cool_name_curr: cool_names[10],
 		cool_name_fairy: cool_names[11],
 		cool_name_dot: cool_names[12],
-    cool_name_doublestruck: cool_names[13],
+    		cool_name_doublestruck: cool_names[13],
```
Comment (mohitkyadav) [inline server.js]: Fix the indentation.

File: services/uncoolifyService.spec.js (hunk 1 above)
Comment (Sumit0581) [reply services/uncoolifyService.spec.js]: Solved the issue in new commit 
https://github.com/NJACKWinterOfCode/cool-name-api/commit/408b5cb26059e14066492ec89f888a441ab6629b

File: services/coolifyService.js (hunk 2 above)
Comment (Sumit0581) [reply services/coolifyService.js]: Fixed the indentation https://github.com/NJACKWinterOfCode/cool-name-api/pull/15/commits/5729fd65abecf22197242ac91504d633a7f9d7f7

**Comment by Sumit0581** [comment]
Commit https://github.com/NJACKWinterOfCode/cool-name-api/pull/15/commits/1b97e1d8d2882b6f6483731ef76dc15b37bf2c2f fixes #16 
Commit https://github.com/NJACKWinterOfCode/cool-name-api/pull/15/commits/bfbb3cf619456defd2a054d54eb0d1ffbeea1b13 fixes #17 
Commit https://github.com/NJACKWinterOfCode/cool-name-api/pull/15/commits/bfaf1d262a241c253a3f8506139f77433e9cc85a chances input field maxlength to accommodate more than 100 characters

File: server.js (hunk 3 above)
Comment (Sumit0581) [reply server.js]: Fixed the indentation https://github.com/NJACKWinterOfCode/cool-name-api/pull/15/commits/5729fd65abecf22197242ac91504d633a7f9d7f7

File: server.js [hunk 4]
```diff
@@ -37,7 +37,8 @@ app.get("/coolify", (request, response) => {
 		cool_name_curr: cool_names[10],
 		cool_name_fairy: cool_names[11],
 		cool_name_dot: cool_names[12],
-    cool_name_doublestruck: cool_names[13],
+    	cool_name_doublestruck: cool_names[13],
```
Comment (mohitkyadav) [inline server.js]: Please fix the indentation.

File: services/coolifyService.js [hunk 5]
```diff
@@ -359,9 +387,10 @@ coolify.alphaNumericName = function(name) {
 		let cool_name_curr = name.allReplace(coolifyDicts.curr);
 		let cool_name_fairy = name.allReplace(coolifyDicts.fairy);
 		let cool_name_dot = name.allReplace(coolifyDicts.dot);
-    let cool_name_doublestruck = name.allReplace(coolifyDicts.doublestruck);
+    	let cool_name_doublestruck = name.allReplace(coolifyDicts.doublestruck);
```
Comment (mohitkyadav) [inline services/coolifyService.js]: Please fix the indentation.

File: server.js (hunk 4 above)
Comment (Sumit0581) [reply server.js]: Fixed indentation issue

File: services/coolifyService.js (hunk 5 above)
Comment (Sumit0581) [reply services/coolifyService.js]: Fixed indentation issue.https://github.com/NJACKWinterOfCode/cool-name-api/pull/15/commits/14caec93d55eafd1e045b65162ad48312bf31377

File: services/coolifyService.js [hunk 6]
```diff
@@ -349,7 +377,7 @@ coolify.alphaNumericName = function(name) {
 		let cool_name_rounded = name.allReplace(coolifyDicts.rounded);
 		let cool_name_square = name.allReplace(coolifyDicts.square);
 		let cool_name_emoji = name.allReplace(coolifyDicts.emoji);
-		let cool_name_round_alphanum = cool_name_rounded.slice(0, 1) + cool_name_alphanum.slice(1, name.length - 1)
+		let cool_name_round_alphanum = cool_name_rounded.slice(0, 1) + cool_name_alphanum.slice(1, name.length - 1)7he qu!(k brown fox jumped over 7he |@zy dog
```
Comment (mohitkyadav) [inline services/coolifyService.js]: I guess you messed up 😄

Comment (Sumit0581) [reply services/coolifyService.js]: screenshot from 2018-12-20 20-37-49
Project successfully running after https://github.com/NJACKWinterOfCode/cool-name-api/pull/15/commits/e4beaaaf32b440e14af330599df6f27d7ccfb1e3

**Pull request merged**